docs/shared/nx-core.md

```diff
@@ -147,14 +164,40 @@ Then add the following to the root `package.json` (which enables Yarn Workspaces
 }
 ```
 
-Finally, run `yarn`.
+// placeholder: this does not work? still get cannot find module '@myorg/simple',
+
+```bash
+> nx run complex:test --verbose
+warning package.json: No license field
+$ node index.js --verbose=true
+node:internal/modules/cjs/loader:936
+  throw err;
+  ^
+
+Error: Cannot find module '@myorg/simple'
+Require stack:
+- /Users/caleb/Sandbox/myorg/packages/complex/index.js
+    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
+    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
+    at Module.require (node:internal/modules/cjs/loader:1005:19)
+    at require (node:internal/modules/cjs/helpers:102:18)
+    at Object.<anonymous> (/Users/caleb/Sandbox/myorg/packages/complex/index.js:1:1)
+    at Module._compile (node:internal/modules/cjs/loader:1101:14)
+    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
+    at Module.load (node:internal/modules/cjs/loader:981:32)
+    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
+    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
+  code: 'MODULE_NOT_FOUND',
+  requireStack: [ '/Users/caleb/Sandbox/myorg/packages/complex/index.js' ]
+}
+```
 
-`yarn nx test complex` works now.
+Finally, run `yarn`, then `yarn nx test complex` works now.
```

Comment:
this still doesn't work. not sure if there is a missing step or another issue, but could not get this guide to work which is not great trying to sell people on nx core

current error 
```bash
warning package.json: No license field
$ node index.js --verbose=true
node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module '@myorg/simple'
Require stack:
- /Users/caleb/Sandbox/myorg/packages/complex/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/caleb/Sandbox/myorg/packages/complex/index.js:1:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/caleb/Sandbox/myorg/packages/complex/index.js' ]
}
```